Extension/src/LanguageServer/extension.ts

```diff
@@ -219,19 +219,22 @@ export async function getBuildTasks(): Promise<vscode.Task[]> {
         // Handle compiler args in compilerPath.
         let compilerPathAndArgs: util.CompilerPathAndArgs = util.extractCompilerPathAndArgs(compilerPath);
         compilerPath = compilerPathAndArgs.compilerPath;
-        const taskName: string = path.basename(compilerPath) + " build active file";
-        let args: string[] = ['-g', '${file}', '-o', path.join('${fileDirname}', '${fileBasenameNoExtension}' + (isWindows ? '.exe' : ''))];
+        const filePath: string = path.join('${fileDirname}', '${fileBasenameNoExtension}');
+        const compilerPathBase: string = path.basename(compilerPath);
+        const taskName: string = compilerPathBase + " build active file";
+        const isCl: boolean = taskName.startsWith("cl.exe");
+        let args: string[] = isCl ? [ '${file}', '/Zi' ] : ['-g', '${file}', '-o', filePath + (isWindows ? '.exe' : '')];
```

Comment:
Do we want to add `/EHsc` and `/Fe:` (like I put in the [pending documentation](https://github.com/mikeblome/vscode-docs/blob/mb-cpp-updates/docs/cpp/cpp-config-msvc.md#create-a-build-task))?

Reply:
I can add /EHsc, but I don't think /Fe is needed, right?

Reply:
Oh, the docs use FE to make the .exe have a different name from the source...should I add anyway, even it even though it's not needed in this case?